server/templates/email/base.html

```diff
@@ -217,7 +219,7 @@
         <table>
           <tr>
             <td align="center">
-              <p> This email was sent to you as a user of <a href="http://okpy.org">okpy.org</a></p>
+              <p> This email was sent to you by a course on <a href="http://okpy.org">okpy.org</a></p>
```

Comment:
How about "as part of a course using" -  

These emails with saying "by a course" (or course staff) is that this applies to partner emails too - and those aren't quite "sent by staff"